src/core/AutomationClip.cpp

```diff
@@ -76,7 +78,9 @@ AutomationClip::AutomationClip( AutomationTrack * _auto_track ) :
 
 AutomationClip::AutomationClip( const AutomationClip & _clip_to_copy ) :
 	Clip( _clip_to_copy.m_autoTrack ),
+#if (QT_VERSION < QT_VERSION_CHECK(5,15,0))
```

Comment:
Needs to match the version check in the header.
```suggestion
#if (QT_VERSION < QT_VERSION_CHECK(5,14,0))
```